service/dynamodb/endpoints.go

```diff
@@ -489,7 +490,7 @@ type endpointParamsBinder interface {
 	bindEndpointParams(*EndpointParameters)
 }
 
-func bindEndpointParams(input interface{}, options Options) *EndpointParameters {
+func bindEndpointParams(input interface{}, options Options, ctx context.Context) *EndpointParameters {
```

Comment:
Shouldn't we be seeing the account ID binding getting generated here? Specifically somewhere from line 496 on?

Reply:
I regenerated the client using model without accountID builtin in my last commit.  I will regenerate that using mocked model